hdfs3/tests/test_example.py

```diff
@@ -0,0 +1,23 @@
+from hdfs3 import HDFileSystem
+
+def test_example():
+    import time
+    import os
+    os.environ["LIBHDFS3_CONF"] = "/etc/hadoop/conf/hdfs-site.xml"
```

Comment:
Looks like it is useful.  Getting short circuit reads warnings now

On Tue, Jan 5, 2016 at 6:34 PM, Daniel Rodriguez <notifications@github.com>
wrote:

> In hdfs3/tests/test_example.py
> <https://github.com/blaze/hdfs3/pull/12#discussion_r48922272>:
>
> > @@ -0,0 +1,23 @@
> > +from hdfs3 import HDFileSystem
> > +
> > +def test_example():
> > +    import time
> > +    import os
> > +    os.environ["LIBHDFS3_CONF"] = "/etc/hadoop/conf/hdfs-site.xml"
>
> I don't think its needed for this test but i its needed for something,
> short circuit reads? I moved it to the Dockerfile
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/blaze/hdfs3/pull/12/files#r48922272>.
>